week2/homework/src/index.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+/** ---------------------------------------
+     Base
+ ----------------------------------------- */
+const program = require('commander');
+const help = require('./intro.js');
+const add = require('./add.js');
+const update = require('./update.js');
+const list = require('./list.js');
+const remove = require('./remove.js');
+const reset = require('./reset.js');
+
+/** ---------------------------------------
+     Commands
+ ---------------------------------------- */
```

Comment:
Great job with the commander here! Don't forget to handle the case when no command is specified (should display the help function)

Reply:
We could use 'node index.js' or 'node index.js help' for showing help section as per assignment. That's why I preffered index.js help. But, I've added commandless option as well.